presto-product-tests/bin/run_on_docker.sh

```diff
@@ -117,8 +117,10 @@ environment_compose up --no-color --abort-on-container-exit ${SERVICES} &
 # Otherwise docker-compose may created network twice and subsequently fail.
 retry check_any_container_is_up
 
-# wait until hadoop processes are started
-retry check_hadoop
+if [[ $SERVICES == *"hadoop-master"* ]]; then
+    # wait until hadoop processes are started
+    retry check_hadoop
+fi
```

Comment:
This is generally OK, but now in Kafka tests you're not waiting for anything.
Previously you waited for hadoop -- hadoop was not needed, but _waiting_ could have been needed.
This may cause flakiness.

How long does it take Kafka to start, compared to Presto startup?
(ie % rather than s)

Reply:
> ow in Kafka tests you're not waiting for anything.

I wait for `Presto`.

> How long does it take Kafka to start, compared to Presto startup?

 Kafka starts an order of magnitude faster.